dandi/metadata.py

```diff
@@ -636,9 +636,7 @@ def extract_session(metadata: dict) -> Optional[List[models.Session]]:
     ]
 
 
-def extract_digest(
-    metadata: dict,
-) -> Optional[Dict[models.DigestType, str]]:
+def extract_digest(metadata: dict) -> Optional[Dict[models.DigestType, str]]:
```

Comment:
I see no content difference -- why is it jumping like this then -- did we change blacking settings since last time this file was modified?

Reply:
No, I just noticed that the signature could fit on one line and felt compelled to fix it.